cypress/integration/save.cy.js

```diff
@@ -52,6 +61,38 @@ describe('save', () => {
         deleteVisualization()
     })
 
+    it('new AO with sorted table saves correctly', () => {
+        const AO_NAME = `TEST SORTING ${new Date().toLocaleString()}`
+        setupTable()
+
+        // save with a name
+        saveVisualization(AO_NAME)
+        expectAOTitleToContain(AO_NAME)
+        expectTableToBeVisible()
+
+        // apply sorting
+        getTableHeaderCells()
+            .find(`button[title*="${TEST_DIM_INTEGER}"]`)
+            .click()
+
+        expectTableToBeUpdated()
+
+        cy.intercept('POST', /eventVisualizations\?/).as('saveAO')
+        cy.intercept('GET', /eventVisualizations\/[a-zA-Z0-9]+\?/).as('loadAO')
+
+        saveVisualizationAs(AO_NAME)
+
+        cy.wait('@saveAO')
+            .its('request.body')
+            .should('have.property', 'sorting')
+
+        cy.wait('@loadAO')
+            .its('response.body')
+            .should('have.property', 'sorting')
+
+        deleteVisualization()
+    })
+
```

Comment:
consider adding a Cypress test for what happens if you try to save a sorting prop that’s not part of the visualisation

Reply:
I don't think that is possible, if it's not part of the visualisation is not saved.
Or do you mean simulate a POST request with "tempered" payload?
(Is this even a valid pattern for Cypress testing?)

Reply:
Added a test for verifying that sorting data is removed when a column with applied sorting is removed from layout.